src/health/azure/himl.py

```diff
@@ -31,16 +32,15 @@
 from health.azure.datasets import (DatasetConfig, StrOrDatasetConfig, _input_dataset_key, _output_dataset_key,
                                    _replace_string_datasets)
 
-logger = logging.getLogger('health.azure')
-logger.setLevel(logging.DEBUG)
-
 RUN_RECOVERY_FILE = "most_recent_run.txt"
 WORKSPACE_CONFIG_JSON = "config.json"
 AZUREML_COMMANDLINE_FLAG = "--azureml"
 RUN_CONTEXT = Run.get_context()
 OUTPUT_FOLDER = "outputs"
 LOGS_FOLDER = "logs"
 AML_IGNORE_FILE = ".amlignore"
+SDK_NAME = "innereye"
```

Comment:
What is the SDK name and version? Isn't it hi-ml? Shouldn't the version be the same as in PyPI?